CHANGE_LOG

```diff
@@ -72,6 +72,7 @@ General Enhancements:
 
 Fixes:
 
+* PR #5092: Fix #5087, bug in bytecode analysis.
```

Comment:
This should be the last item? PR numbers are ordered descending?

Reply:
oops, you're right. confused this with the meeting notes

Reply:
fixed